### PR TITLE
Profile card

### PR DESCRIPTION
**Pull request by janvi01**
## Related Issue
  - fixes issue #736 

Closes: #736 

### Describe the changes you've made
Added a Profile Card Generator.

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.

Please review my code.

### CODE REVIEW

**Comment by 991rajat** [comment]
@janvi01  Add the project to script.js.

**Comment by janvi01** [comment]
Is it okay now? @991rajat

File: script.js [hunk 1]
```diff
@@ -234,6 +234,152 @@ let projectData = [
     projectImage: "assets/GIFs/line-replacing-project-.gif",
     projectUrl: "public/line_replacing_project.html",
   },
+  {
```
Comment (991rajat) [inline script.js:237]: Only Add you project. You have added other projects again too. Fix this please

**Review by 991rajat — CHANGES_REQUESTED** [review]
image
Make the buttons horizontally parallel

**Comment by janvi01** [comment]
is it okay now? @991rajat

**Comment by 991rajat** [comment]
Could you please fix script.js too.
Add only your project.

**Comment by janvi01** [comment]
I have added in previous commit only, 
please check @991rajat

**Comment by 991rajat** [comment]
@janvi01 Remove line 237-381
and add this insted.
 {
    projectName: "Profile Card Generator",
    projectImage: "assets/Images/Profilecard.png",
    projectUrl: "public/profilecard.html",
  },
  
  It will fix it.

**Comment by janvi01** [comment]
I am making new PR, sorry for the inconvenience

**Comment by 991rajat** [comment]
NO problem.

**Pull request closed without merging**